packages/local_auth/README.md

```diff
@@ -142,6 +142,12 @@ Update your project's `AndroidManifest.xml` file to include the
 <manifest>
 ```
 
+On Android, you can only check for existence of fingerprint hardware prior
+to API 29 (Android Q). Therefore, if you would like to support other biometrics
+types (such as face scanning), *do not* call `getAvailableBiometrics`. Simply call
+`authenticateWithBiometrics`. This will return an error if there was no hardware
```

Comment:
I think it would be better to have a consistent Dart API here. Instead of asking plugin users to check their SDK and then call plugin APIs differently, could that SDK/Platform check be handled from within the plugin itself so `getAvailableBiometrics` consistently returns the right data and users don't need to use Error handling for their control flow? From what I can tell SDK 29 should still be able to get this information using [`hasSystemFeature(FEATURE_FACE)`](https://developer.android.com/reference/android/content/pm/PackageManager.html#FEATURE_FACE), but maybe I'm missing something here? In general it's not unreasonable to case off of SDK versions within the plugin Java code.

Reply:
Unfortunately, there are phones out there with face unlock which are on API 27 or 28. While androidx.biometric package works for these, there's no way of knowing that the phones have that hardware without actually instantiating authentication process :(.

The best I can do here is:
- Return a definitive answer for API >= 29
- Return a definitive answer only for fingerprint and a "may be" answer for API < 29.

IOW, this API is not usable below 29.

Reply:
Ah, I see, thanks for the clarification. :( Strange that `androidx.biometric` doesn't have any getters when it adds functionality on earlier SDKs.

Even with that in mind, I think it would be cleaner for this to give as much of a definitive answer as it can, so returning `FEATURE_FACE` on 29 and above. Then this note in the README would only be relevant for callers who specifically want to go beyond and use face unlock < 29.

Reply:
Well androidx.biometric is still alpha. Not all functionality is there. :fearful: 

The change you are requesting is reasonable. I will also polish the wording here a bit more.

Reply:
This is done. PTAL.